SPECS/core-packages/core-packages.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Metapackage with core sets of packages
 Name:           core-packages
 Version:        3.0
-Release:        10%{?dist}
```

Comment:
Just making sure, is release `2` going to be picked up? I understand it should have been release `1` instead of `10`.

Reply:
Previous value was just accidentally left at 10 after a version change. Current 2 is correct.